aquadoggo/src/materializer/worker.rs

```diff
@@ -324,6 +349,11 @@ where
         self.error_handle.clone()
     }
 
+    /// Subscribe to status changes of tasks.
+    pub fn on_update(&self) -> Receiver<TaskStatus<IN>> {
+        self.tx_status.subscribe()
+    }
+
```

Comment:
The name of this method threw me off a little, it doesn't quite describe it's function for me. I would expect something like `on_task_status_change`. I'll get used to it if there's a reason for the current naming though :+1:

Reply:
Good idea!